packages/dbus_glib.rb

```diff
@@ -0,0 +1,25 @@
+require 'package'
+
+class Dbus_glib < Package
+  description 'The D-Bus GLib package contains GLib interfaces to the D-Bus API.'
+  homepage 'http://www.linuxfromscratch.org/blfs/view/svn/general/dbus-glib.html'
+  version '0.110'
+  source_url 'https://dbus.freedesktop.org/releases/dbus-glib/dbus-glib-0.110.tar.gz'
+  source_sha256 '7ce4760cf66c69148f6bd6c92feaabb8812dee30846b24cd0f7395c436d7e825'
+
+  binary_url ({
+  })
+  binary_sha256 ({
+  })
+
+  depends_on 'gtk_doc'
```

Comment:
You are missing `dbus` and  `glib` dependencies.  Test on a fresh crew install.

Reply:
OK.

Reply:
LOL, that makes sense.

Reply:
Remove the `depends_on 'gtk_doc'` dependency?

Reply:
OK.

Reply:
Can we not just remove the gtk_doc dependency here and submit the docbook madness in another PR please?

Reply:
I can't untrack them now. There will be merge conflicts.